core/src/main/java/org/opentosca/toscana/core/csar/Csar.java

```diff
@@ -0,0 +1,20 @@
+package org.opentosca.toscana.core.csar;
+
+import org.opentosca.toscana.core.transformation.TransformationImpl;
+
+import java.util.Map;
+
+public interface Csar {
+
+    /**
+     * Returns a map of all transformation objects of this csar.
+     * This includes scheduled, ongoing and finished transformations.
+     * Key of each map entry is the platform identifier of its particular transformation.
+     */
+    public Map<String,TransformationImpl> getTransformations();
+
+    /**
+     * Returns the identifier of the CSAR (which serves as identifier)
+     */
+    public String getIdentifier();
```

Comment:
We also need a way to read the archive in this interface i would say something like:
```Java
public InputStream getCsarInputStream() throws IOException;
```

Reply:
I thought about:
- forwarding the inputstream to CsarDao
- CsarDao persists archive to disk
- CsarDao unzips archive (maybe also deletes original archive to save disc space). [unsure about this step: today i try to integrate opentosca's parser, then i might know more]
- CsarDao gives File references to Csar. We don't want to work with inputstreams but use the File API.

Keep in mind this is all [WIP]